dask/dataframe/groupby.py

```diff
@@ -1008,6 +1008,9 @@ def _build_agg_args_single(result_column, func, func_args, func_kwargs, input_co
     elif func == "list":
         return _build_agg_args_list(result_column, func, input_column)
 
+    elif func == "collect":
```

Comment:
Could we check that the user is using dask-cudf and not dask with pandas when getting here?

Reply:
Any suggestions on a clean way to do this? It doesn't seem like any type information should be passed to `_build_agg_args_single`?

Reply:
In case it helps, trying to pass `"collect"` for  pandas-based data will result in a reasonable metadata error:

```
RuntimeError: Failed to generate metadata for DecomposableGroupbyAggregation(frame=df, arg='collect', observed=False, split_out=1). This operation may not be supported by the current backend.
```

Reply:
Perhaps it will be easy-enough to convert "collect" to a proper `Aggregation` in dask-cudf... Will give that a try first.